PROJECT

```diff
@@ -1,6 +1,6 @@
 domain: openshift.io
 layout:
-- go.kubebuilder.io/v3
+- go.kubebuilder.io/v4
```

Comment:
I'm not thrilled about migrating to the new layout (https://book.kubebuilder.io/migration/manually_migration_guide_gov3_to_gov4#changes-to-the-layout) in a future PR of course, but I won't oppose it either.

Reply:
> I'm not thrilled about migrating to the new layout (https://book.kubebuilder.io/migration/manually_migration_guide_gov3_to_gov4#changes-to-the-layout) in a future PR of course, but I won't oppose it either.

It can be removed from the current PR.
The reason I added it here is because kubebuilder is complaining about the fact that gov3 is deprecated

Reply:
Yes, let's keep it v3 for now to reflect the actual laout of the project